tests/js/sudoku.scrypttest.js

```diff
@@ -0,0 +1,125 @@
+const { expect } = require('chai');
+const { buildContractClass, getPreimage, toHex, num2bin, Bytes } = require('scryptlib');
+
+const {
+  newTx,
+  compileContract
+} = require('../../helper');
+
+
+
+const solution = [
+
+  [2,1,9,4,8,6,7,5,3],
+  [7,4,5,9,1,3,2,8,6],
+  [8,3,6,7,5,2,1,4,9],
+  [5,6,3,2,7,4,9,1,8],
+  [4,8,7,1,3,9,6,2,5],
+  [9,2,1,5,6,8,3,7,4],
+  [6,7,2,3,4,5,8,9,1],
+  [3,9,4,8,2,1,5,6,7],
+  [1,5,8,6,9,7,4,3,2]
+];
+
+
+// board, 0 is empty value
+const board = [
```

Comment:
Let's have a "sparse" board, i.e., more `0`s, to be realistic.

Reply:
E.g.
https://en.wikipedia.org/wiki/Sudoku#Variations_of_grid_sizes_or_region_shapes
https://rosettacode.org/wiki/Category:BBC_BASIC